.github/workflows/tests-l4lb-v1.10.yaml

```diff
@@ -183,7 +191,7 @@ jobs:
 
       - name: Run tests
         run: |
-          ssh default "sudo /bin/sh -c 'cd /vagrant/test/l4lb && ./test.sh ${{ github.repository_owner}} ${{ steps.vars.outputs.sha }}'"
+          ssh default "sudo /bin/sh -c 'cd /vagrant/test/l4lb && ./test.sh ${{ github.repository_owner}} ${{ steps.vars.outputs.sha }} ${{ github.workspace }}/pull-request'"
```

Comment:
Don't we need to use `test.sh` from the PR checkout as well? Or otherwise we'd need to change the script in `v1.10` and `v1.11` branch to take the third argument with the helm chart directory as well.

Reply:
Please disregard, I just saw https://github.com/cilium/cilium/pull/19953#discussion_r881793284. I think we still need to change `test.sh` in `v1.10` and `v1.11` to take the third argument, though.